Java/Data Type & Operators/1. Hello World/vatsal.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
```

Comment:
rename the file with capital firt letter
and keep the file name and class name same